lib/embedded_ansible.rb

```diff
@@ -60,13 +61,19 @@ def self.start
   end
 
   def self.stop
-    return if MiqEnvironment::Command.is_container?
-    services.each { |service| LinuxAdmin::Service.new(service).stop }
+    if MiqEnvironment::Command.is_container?
+      container_stop
+    else
+      services.each { |service| LinuxAdmin::Service.new(service).stop }
```

Comment:
I'm wondering if this should be extracted to a method like the container stuff is extracted to a method.  Then this method just becomes a simple redirection method with no logic.

```ruby
def self.stop
  MiqEnvironment::Command.is_container? ? container_stop : services_stop
end
```

Upon reading that maybe the methods should be `stop_containers` and `stop_services`, so that all 3 stop methods "group" themselves together